certora/spec/timelockAuthorizerTed.spec

```diff
@@ -173,7 +187,7 @@ rule scheduleRootChangeCreatesSE(env e) {
     require(numberOfSchedExeBefore < max_uint / 4);
 
     scheduleRootChange@withrevert(e, newRoot, executors);
-
+// isn't the withrevert redundant here? 
```

Comment:
Yes, you are right: In case it reverts, the tool does not check the assertions. We can remove the implications and write this:
```
assert numberOfSchedExeAfter == numberOfSchedExeBefore + 1;
assert rootBefore == _root();
assert pendingRootBefore == getPendingRoot();
```